doc/usage/other-clients.rst

```diff
@@ -17,3 +17,8 @@ Emacs Interface
 ---------------
 
 Github: `phpactor.el <https://github.com/emacs-php/phpactor.el>`_
+
+Visual Studio Code
+------------------
+
+Github: `vscode-phpactor <https://marketplace.visualstudio.com/items?itemName=williambrook.phpactor>`_
```

Comment:
this is not the official VS code extension (it's an extension to the leagcy RPC), the official LSP extension is: https://github.com/phpactor/vscode-phpactor